packages/grafana-ui/src/components/Logs/getLogRowStyles.ts

```diff
@@ -1,10 +1,10 @@
 import { css } from '@emotion/css';
 import { GrafanaTheme2, LogLevel } from '@grafana/data';
-import { styleMixins, stylesFactory } from '../../themes';
+import { stylesFactory } from '../../themes';
 
 export const getLogRowStyles = stylesFactory((theme: GrafanaTheme2, logLevel?: LogLevel) => {
   let logColor = theme.isLight ? theme.v1.palette.gray5 : theme.v1.palette.gray2;
-  const hoverBgColor = styleMixins.hoverColor(theme.colors.background.primary, theme);
+  const hoverBgColor = 'rgba(215, 229, 255, 0.4)';
```

Comment:
Can we pull these types of changes out into a `Fusebit` theme file somewhere and just change the current theme?  Or "own" the `isLight` theme and just make changes in that file?

Anything we can do to reduce code changes that'll make our lives hard when we eventually upgrade will be appreciated by future us.

Reply:
Okey, i'll change the colors from the theme instead of hardcoding them